test/acceptance/features/steps/steps.py

```diff
@@ -276,3 +276,50 @@ def then_envFrom_contains(context, app_name, sbr_name1, sbr_name2):
 def then_envFrom_contains_intermediate_secret_name(context, intermediate_secret_name):
     assert context.application.get_deployment_with_intermediate_secret(
         intermediate_secret_name, wait=True, timeout=120) is not None, f"There is no deployment with intermediate secret {intermediate_secret_name}"
+
+
+# STEP
+@given(u'Backend infrastructure is running')
+def given_backend_infrastructure(context):
+    openshift = Openshift()
+    _ = openshift.oc_apply_yaml_file(os.path.join(os.getcwd(), "test/operator-hub/backend-csv.yaml"))
+    _ = openshift.oc_apply_yaml_file(os.path.join(os.getcwd(), "test/third-party-crds/backend-crd.yaml"))
+
+
+# STEP
+@given(u'Backend CR "{cr_name}" is applied')
+def create_backend_cr(context, cr_name):
+    openshift = Openshift()
+    yaml = context.text
+    output = openshift.oc_apply(yaml)
+    result = re.search(rf'.*{cr_name}.*(created|unchanged)', output)
+    result | should_not.be_none.desc("CRD {cr_name} Created")
```

Comment:
you can reuse existing function and replace this with:

```
def create_backend_cr(context, cr_name):
   create_cr(context, cr_name)
```

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/597/commits/2899fd0b882ae30ee3b19d149d9329703c878e0c